static/app/views/settings/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -331,28 +331,46 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
     const hasIntegration = configurations ? configurations.length > 0 : false;
     const endpoint = `/organizations/${organization.slug}/`;
     const hasOrgWrite = organization.access.includes('org:write');
-
+    let openPRDisabledReason = t(
+      'You must have a GitHub integration to enable this feature.'
+    );
+    if (!organization.features.includes('integrations-open-pr-comment')) {
+      openPRDisabledReason = t("This feature isn't available to you yet.");
+    }
     const forms: JsonFormObject[] = [
       {
         fields: [
           {
             name: 'githubPRBot',
             type: 'boolean',
-            label: t('Enable Pull Request Bot'),
+            label: t('Enable Comments on Suspect Pull Requests'),
             help: t(
-              'Allow Sentry to comment on pull requests about issues impacting your app.'
+              'Allow Sentry to comment on recent pull requests suspected of causing issues.'
             ),
             disabled: !hasIntegration,
             disabledReason: t(
               'You must have a GitHub integration to enable this feature.'
             ),
           },
+          {
+            name: 'githubOpenPRComments',
+            type: 'boolean',
+            label: t('Enable Comments on Open Pull Requests'),
+            help: t(
+              'Allow Sentry to comment on open pull requests to show recent error and performance issues for the code being changed.'
+            ),
+            disabled:
+              !hasIntegration ||
+              !organization.features.includes('integrations-open-pr-comment'),
```

Comment:
instead of checking for the feature here, you can add another field to this blob like this
```typescript
visible: ({features}) => features.has('integrations-open-pr-comment')
```
this hides the toggle entirely if the org doesn't have the feature flag

then you can pass features into the JsonForm
```typescript
<JsonForm features={organization.features} forms={forms} />
```